backend/de.metas.util.web/src/main/java/de/metas/util/web/filter/ApiAuditFilter.java

```diff
@@ -105,10 +116,26 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
 		{
 			logger.error(t.getLocalizedMessage(), t);
 
-			apiAuditService.handleErrorResponse(null, t, httpServletResponse);
+			apiAuditService.handleErrorResponse(t, httpServletResponse, null, null);
 		}
 	}
 
+	private void auditHttpCallAsync(
+			@NonNull final HttpServletRequest httpServletRequest,
+			@NonNull final HttpServletResponse httpServletResponse,
+			@NonNull final FilterChain chain) throws IOException, ServletException
+	{
+		final ContentCachingResponseWrapper contentCachedResponse = new ContentCachingResponseWrapper(httpServletResponse);
+		final ContentCachingRequestWrapper contentCachedRequest = new ContentCachingRequestWrapper(httpServletRequest);
+
+		chain.doFilter(contentCachedRequest, contentCachedResponse);
```

Comment:
we can expect to have out catch-all configs, so i think we should fail if there is no config.
To make it clear that there is no such contingency and thus reduce FUD